bundles/org.openhab.voice.voicerss/README.md

```diff
@@ -25,13 +29,135 @@ https://api.voicerss.org/?key=API_KEY&hl=de-de&f=44khz_16bit_mono&src=Hallo%20We
 
 ## Configuration
 
-You must add your API_KEY to your configuration by adding a file "voicerss.cfg" to the services folder, with this entry:
+The following configurations can be edited using the "VoiceRSS Text-to-Speech" settings in the UI:
+
+* **VoiceRSS API Key** - The API Key to get access to http://www.voicerss.org.
+
+In case you would like to setup the service via a text file, create a new file in `$OPENHAB_ROOT/conf/services` named `voicerss.cfg`
+
+Its contents should look similar to:
 
 ```
-apiKey=1234567890
+org.openhab.voice.voicerss:apiKey=1234567890
 ```
 
-It actually supports only one voice: "voicerss:default", which is configured to use 44kHz, mono, 16 bit sampling quality.
+## Voices
+
+It actually supports the following voices (depending on language):
```

Comment:
I think you meant "currently" (I know I'm french too :-) )

Reply:
It is a copy/paste from another README! 
I hesitated to update it for that reason. 
I will fix there and run a great in other bindings.

Reply:
Removed